<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>0</y>
      <width>715</width>
      <height>490</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Principal</string>
     </attribute>
     <widget class="QPushButton" name="botao_adicionar">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>175</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Idade:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nome">
      <property name="geometry">
       <rect>
        <x>89</x>
        <y>9</y>
        <width>159</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idade">
      <property name="geometry">
       <rect>
        <x>89</x>
        <y>47</y>
        <width>159</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabela">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>90</y>
        <width>693</width>
        <height>275</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="statusTip">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>70</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>93</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Idade</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Peso</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dist창ncia</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tempo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Velocidade</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Gasto Cal처rico</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="botao_ordenar_nome">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>402</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botao_ordenar_idade">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>402</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Idade</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botao_ordenar_peso">
      <property name="geometry">
       <rect>
        <x>175</x>
        <y>402</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Peso</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_distancia">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>9</y>
        <width>85</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>12</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Dist창ncia:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botao_ordenar_velocidade">
      <property name="geometry">
       <rect>
        <x>435</x>
        <y>402</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Velocidade</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botao_ordenar_tempo">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>402</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Tempo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botao_ordenar_distancia">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>402</y>
        <width>85</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Dist창ncia</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botao_ordenar_calorias">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>402</y>
        <width>135</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Calorias Gastas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>282</x>
        <y>12</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Peso:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_peso">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>9</y>
        <width>159</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>282</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tempo:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_tempo">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>47</y>
        <width>159</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>375</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ordenar:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botao_limpar_tabela">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>370</y>
        <width>135</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpar Tabela</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <attribute name="title">
      <string>Estat챠sticas</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>660</width>
        <height>145</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>M챕dia</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>25</y>
         <width>221</width>
         <height>36</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Dist창ncia</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>25</y>
         <width>221</width>
         <height>36</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Tempo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>25</y>
         <width>220</width>
         <height>36</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Velocidade</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_med_distancia">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>220</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_med_tempo">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>220</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_med_velocidade">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>60</y>
         <width>220</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>660</width>
        <height>145</height>
       </rect>
      </property>
      <property name="title">
       <string>Destaques</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>25</y>
         <width>221</width>
         <height>36</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Maior Dist창ncia Percorrida</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_maior_distancia">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>220</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_menor_tempo_gasto">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>220</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_mais_veloz">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>60</y>
         <width>220</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>25</y>
         <width>221</width>
         <height>36</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Menor Tempo Gasto</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>25</y>
         <width>220</width>
         <height>36</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Mais Veloz</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>660</width>
        <height>145</height>
       </rect>
      </property>
      <property name="title">
       <string>Desvio Padr찾o</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>25</y>
         <width>221</width>
         <height>36</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Dist창ncia</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>25</y>
         <width>221</width>
         <height>36</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Tempo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>25</y>
         <width>220</width>
         <height>36</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Velocidade</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_dp_distancia">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>220</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_dp_tempo">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>220</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_dp_velocidade">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>60</y>
         <width>220</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="Salvar"/>
    <addaction name="Carregar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Salvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="Carregar">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_nome</tabstop>
  <tabstop>lineEdit_idade</tabstop>
  <tabstop>lineEdit_peso</tabstop>
  <tabstop>lineEdit_tempo</tabstop>
  <tabstop>lineEdit_distancia</tabstop>
  <tabstop>botao_adicionar</tabstop>
  <tabstop>botao_ordenar_nome</tabstop>
  <tabstop>botao_ordenar_idade</tabstop>
  <tabstop>botao_ordenar_peso</tabstop>
  <tabstop>botao_ordenar_distancia</tabstop>
  <tabstop>botao_ordenar_tempo</tabstop>
  <tabstop>botao_ordenar_velocidade</tabstop>
  <tabstop>botao_ordenar_calorias</tabstop>
  <tabstop>botao_limpar_tabela</tabstop>
  <tabstop>tabela</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
